ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/_cq_dialog/.content.xml

```diff
@@ -69,10 +69,53 @@
                                                     sling:resourceType="granite/ui/components/coral/foundation/form/hidden"
                                                     name="./type"/>
                                             <enums jcr:primaryType="nt:unstructured"
-                                                   sling:resourceType="granite/ui/components/coral/foundation/include"
-                                                   path="core/fd/components/form/base/v1/base/cq:dialog/content/items/tabs/items/basic/items/columns/items/column/items/enums"
-                                                   sling:orderBefore="placeholder"
-                                                   sling:hideResource="{Boolean}false"/>
+                                                   sling:resourceType="granite/ui/components/coral/foundation/container"
+                                                   sling:orderBefore="bindref">
+                                                <items jcr:primaryType="nt:unstructured">
+                                                    <enumOptions jcr:primaryType="nt:unstructured"
+                                                                 sling:orderBefore="bindref"
+                                                                 sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                                                 fieldDescription="Provide the pair of enum (data value) and enumName (display text) for each option."
+                                                                 fieldLabel="Options">
+                                                        <field jcr:primaryType="nt:unstructured"
+                                                               sling:resourceType="granite/ui/components/coral/foundation/container"
+                                                               name="./enum">
```

Comment:
Container does not have name

Reply:
can't be used, it gives each multifield the same class but not one as a whole